core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMasterProcess.java

```diff
@@ -73,15 +73,22 @@ protected FaultTolerantAlluxioMasterProcess(JournalSystem journalSystem,
   public void start() throws Exception {
     mRunning = true;
     mJournalSystem.start();
+
+    startMasters(false);
+    LOG.info("Secondary started");
+
+    if (ServerConfiguration.getBoolean(PropertyKey.MASTER_JOURNAL_INITIAL_REPLAY_ENABLED)) {
+      mJournalSystem.waitForInitialReplay();
+      LOG.info("Journal initial replay finished");
```

Comment:
(nit) This log seems redundant to line 170 in `UfsJournalSystem`.

Reply:
Removed, thanks